File: 1-codes/micro/micro01.c
#import<stdio.h>

int main(){
	float cel, far;
	cel = 0.0;
  far = 0.0;
  printf("Tabela de conversao: Celsius -> Fahrenheit");

  printf("Digite a temperatura em Celsius: ");
  scanf("%f", &cel);

  far = (9.0 * cel + 160.0) / 5.0;
  printf("%f", far);

	return 0;
}

File: 1-codes/micro/micro01.py
def micro01():
    cel , far = 0.0 , 0.0
    print(" Tabela de conversao: Celsius -> Fahrenheit")
    print("Digite a temperatura em Celsius: ")
    cel = input()
    far = (9 * cel + 160) / 5
    print("A nova temperatura e:" + str(far) + "F")

micro01()
